src/api/bulk.ts

```diff
@@ -1330,6 +1330,7 @@ export class IngestJobV2<
           externalIdFieldName: this.jobInfo?.externalIdFieldName,
           object: this.jobInfo?.object,
           operation: this.jobInfo?.operation,
+          lineEnding: this.jobInfo?.lineEnding,
```

Comment:
What happens if the caller doesn't specify a line ending? docs says it defaults to `LF` if it's not specified, does the same happens if the field in the body passed `undefined`?

Reply:
I can default it based on platform

Reply:
I'm not sure how webpack handles the `process.env` object so not sure what's needed for it to replace it in the browser bundle. Maybe just default the param to `LF`?

Reply:
@cristiand391 it's using [envify](https://github.com/jsforce/jsforce/blob/master/gulpfile.babel.js#LL106C8-L106C14)

Reply:
@cromwellryan that's the `master`/v1 branch, I don't see that in the  `2.0` branch:
https://github.com/jsforce/jsforce/tree/2.0

Reply:
We tested this by setting `lineEnding = undefined` and the server ignores it and defaults to `LF` as expected.